### PR TITLE
Adding the support of iTerm2

### PR DESCRIPTION
**Pull request by jstoja**
Hello,

We discussed about it here https://github.com/boot2docker/boot2docker/issues/414
I kept the support of the default terminal, and let the commands at the top so it will be easier to change them if needed.
Working on my laptop so... Try it out too :)
(btw I didn't signed the commit since it's not asked for this repo ;))

ping @SvenDowideit @jlecour

### CODE REVIEW

**Comment by SvenDowideit** [comment]
mmm, @artiom @jstoja  it still isn't working for me.

If I uninstall iTerm, then running this new script (or even just opening it in the applescript editor) will pop up a `find iTerm` dialog window.

argh?

File: mpkg/boot2dockerutils.pkg/start.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+b2d="/usr/local/bin/boot2docker"
+unset DYLD_LIBRARY_PATH ; unset LD_LIBRARY_PATH
+mkdir -p ~/.boot2docker
+if [ ! -f ~/.boot2docker/boot2docker.iso ]; then cp /usr/local/share/boot2docker/boot2docker.iso ~/.boot2docker/ ; fi
+$b2d ip
+$b2d init
+$b2d up && export DOCKER_HOST=tcp://$(/usr/local/bin/boot2docker ip 2>/dev/null):2376
```
Comment (SvenDowideit) [inline mpkg/boot2dockerutils.pkg/start.sh]: can you change the `export ...` to calling `$(boot2docker shellinit)` - that will automatically set up the entire env, TLS or identity auth, or whatever)

**Comment by artiom** [comment]
@jstoja if you look at my version of the main.scpt: https://github.com/artiom/osx-installer/blob/iterm2_support/mpkg/boot2docker.app/Contents/Resources/Scripts/main.scpt

It loads the whole ```run_iterm``` into a string ```itermCmd``` and than executes it through ```run script itermCmd```. In this PR it still tries to communicate directly with iTerm though ```tell application "iTerm.app"``` as @SvenDowideit mentioned above. Unfortunately, this brings iTerm search prompt.

Just to confirm what @SvenDowideit discovered as well, I get application search prompt too when iTerm is not installed.

**Comment by tianon** [comment]
Sorry, this repository is long-since deprecated in favor of [Docker Toolbox](https://github.com/docker/toolbox) (whose usage is now also discouraged in favor of [Docker for Windows](https://www.docker.com/docker-windows) and [Docker for Mac](https://www.docker.com/docker-mac)).

**Pull request closed without merging**